tests/e2e/common/kubernetes.go

```diff
@@ -164,27 +165,38 @@ func CleanupNode(ctx context.Context, cs clientset.Interface) {
 	}
 }
 
-func CleanupNodeFeatureRules(ctx context.Context, cli *nfdclient.Clientset, namespace string) {
-	// Drop NodeFeatureRule objects
+func CleanupNodeFeatures(ctx context.Context, cli *nfdclient.Clientset, namespace string) {
+	// Clean NodeFeatureRule objects
 	nfrs, err := cli.NfdV1alpha1().NodeFeatureRules().List(ctx, metav1.ListOptions{})
```

Comment:
Question: Don't we have to do the same thing here? Does `List` return `ErrNotFound` if there are no nodeFeatureRules?

Reply:
umm, it is possible, yes